src/components/statusHelper.js

```diff
@@ -343,6 +345,87 @@ export const styledConnectionStatus = (status, err) =>
         </Text>
       </TextContent>
     ),
+    no_rhc: (
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Cannot remediate - Cloud connector not defined
+          <Text component={TextVariants.small} style={{ margin: '0px' }}>
+            Remediation from Insights requires Cloud connector. Cloud connector
+            can be enabled via Satelite, or through
+            <Button
+              style={{ padding: '0px' }}
+              key="configure"
+              variant="link"
+              // eslint-disable-next-line max-len
+              href="LINK HERE"
+            >
+              RHC (Red Hat connector)
+            </Button>
+          </Text>
+          <Button
+            style={{ padding: '0px' }}
+            key="download"
+            variant="link"
+            component="a"
+            // eslint-disable-next-line max-len
+            href="LINK HERE"
+          >
+            Learn how to configure
+          </Button>
+        </Text>
+      </TextContent>
+    ),
+    no_smart_management: (
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Cannot remediate - Not entitled
+          <Text component={TextVariants.small} style={{ margin: '0px' }}>
+            Remediation from Insights is supported only for systems with Cloud
+            connector, a feature of Smart Management
+          </Text>
+          <Button
+            style={{ padding: '0px' }}
+            key="download"
+            variant="link"
+            component="a"
+            // eslint-disable-next-line max-len
+            href="LINK HERE"
+          >
+            Learn about Smart Management
+          </Button>
+        </Text>
+      </TextContent>
+    ),
+    disabled: (
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Cannot remediate - Cloud connector not defined
+          <Text component={TextVariants.small} style={{ margin: '0px' }}>
+            Remediation from Insights requires Cloud connector. Cloud connector
+            can be enabled via Satelite, or through
+            <Button
+              style={{ padding: '0px' }}
+              key="configure"
+              variant="link"
+              // eslint-disable-next-line max-len
+              href="LINK HERE"
+            >
+              RHC (Red Hat connector)
+            </Button>
+          </Text>
+          <Button
+            style={{ padding: '0px' }}
+            key="download"
+            variant="link"
+            component="a"
+            // eslint-disable-next-line max-len
+            href="LINK HERE"
```

Comment:
NOTE:  This link will not be available until the 18th of May as per a discussion with the BU.